cgi-bin/DW/External/Site.pm

```diff
@@ -96,7 +130,8 @@ sub get_site {
                 map { lc $_ }
                 split( /\./, $site );
 
-    return $domaintosite{"$parts[-2].$parts[-1]"} || DW::External::Site::Unknown->accepts( \@parts ) || undef;
+    return $domaintosite{"$parts[-2].$parts[-1]"} || $domaintosite{"$parts[-1]"} || 
```

Comment:
PS: I think there might be a suggestion for something like this. If you have a quick look-through and see, please also include the suggestions link in the issue that you open!